docs/reference-manual/native-image/guides/build-and-run-native-executable-with-remote-jmx.md

```diff
@@ -13,6 +13,12 @@ Remote management using [Java Management Extensions (JMX)](https://www.oracle.co
 
 This guide covers the steps required to build, run, and interact with such a native executable using JMX.
 It also shows you how to register a custom managed bean (MBean), with the JMX server and the additional steps required for it to work with Native Image.
+## Currently Supported Features and Limitations
+Currently, JMX connection from a client to a remote MBean server is supported. The client, server, or both may be a native executable.
```

Comment:
One sentence per line makes it easier to manage such markdowns in git. I'll take care of that in a cleanup commit.